docs/draft/3-whisper-usage.md

```diff
@@ -58,7 +56,7 @@ encryption properties to support asynchronous chat.
 * *Whisper network*: a group of Whisper nodes connected together through the internet connection and forming a graph
 * *Message*: decrypted Whisper message
```

Comment:
missing

Reply:
Hey @oskarth, I'm sorry, I don't understand what is missing. Could you give me some more details? Thank you.

Reply:
a decrypted